src/lumigo_tracer/event/event_trigger.py

```diff
@@ -197,6 +197,10 @@ def _parse_streams(event: dict) -> Dict[str, str]:
         result.update(_parse_sqs_event(event))
     elif triggered_by == "kinesis":
         result["messageId"] = safe_get(event, ["Records", 0, "kinesis", "sequenceNumber"])
+        result["recordsNum"] = len(safe_get(event, ["Records"]))
```

Comment:
@saartochner in case the list is very long (10K) will `len` performance be affected?

Reply:
in python, list's `len` is O(1)